src/Sema.zig

```diff
@@ -11896,16 +11832,16 @@ fn resolveBuiltinTypeFields(
     return sema.resolveTypeFields(block, src, resolved_ty);
 }
 
-fn analyzeStructFields(
-    sema: *Sema,
-    block: *Scope.Block,
+fn semaStructFields(
+    mod: *Module,
     struct_obj: *Module.Struct,
 ) CompileError!void {
```

Comment:
I considered moving `semaStructFields` and `semaUnionFields` into `Module.zig`, but they call a lot of functions that are currently private to `sema` and there isn't a clean place to split this function because it retains a lot of locals.